packages/mds-agency/utils.ts

```diff
@@ -146,10 +147,11 @@ export async function getVehicles(
       throw new Error('device in DB but not in cache')
     }
     const event = eventMap[device.device_id]
+    const event_types = event ? event.event_types : ['decommissioned']
```

Comment:
What does `'decommissioned'` mean here?

Reply:
Our fallback if there's a cache miss is to essentially just say the the device was decommissioned, because we clear the cache entries for a device when we receive an event with `event_types: ['decommissioned']`.

Reply:
And if that's not true, what do we do if we get a trip_start or something?

Reply:
We update the cache with the event info in that case! Then, when it's fetched here, we'll match the condition in the ternary and return the event_types that were on it.